event/src/main/java/net/md_5/bungee/event/EventBus.java

```diff
@@ -55,6 +57,15 @@ public void post(Object event)
                 {
                     logger.log( Level.WARNING, MessageFormat.format( "Error dispatching event {0} to listener {1}", event, method.getListener() ), ex.getCause() );
                 }
+
+                long elapsed = System.currentTimeMillis() - start;
+                if ( elapsed > 200 )
+                {
+                    logger.log( Level.WARNING, "Plugin listener {0} took {1}ms to process event {2}!", new Object[]
+                    {
+                           method.getListener().getClass().getName(), elapsed / 1000000, event
```

Comment:
you're dividing milliseconds here and telling its milliseconds, are we getting days timestamp?

Reply:
Copypaste mistake